src/styles/blocks/card.scss

```diff
@@ -0,0 +1,127 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start ;
+  align-items: center;
+
+  width: 200px;
+  padding: 32px 16px 16px;
+  margin-bottom: 50px;
```

Comment:
BEM-block shouldn't have margins

Reply:
changed all margins to gap. in browser it works, but some tests are failed and it doesn't show those gaps

Reply:
You can use margins, but they should be given to class of BEM-element, not BEM-block